storage/cloudspanner/log_storage.go

```diff
@@ -536,11 +536,14 @@ func (tx *logTX) DequeueLeaves(ctx context.Context, limit int, cutoff time.Time)
 		rows = genDequeueRowsOld(tx.getLogStorageConfig(), tx.treeID, now.Unix(), rand.Int31n(numByteValues))
 	}
 	ret := make([]*trillian.LogLeaf, 0, limit)
+	readOpts := &spanner.ReadOptions{Limit: limit}
+	drain := 0
```

Comment:
'discarded' maybe? Wasn't immediately clear what it meant.

Reply:
I'll remove it - I was just verifying that it did what I expected.